app/backend/approaches/chatreadretrieveread.py

```diff
@@ -23,9 +23,9 @@ class ChatReadRetrieveReadApproach(Approach):
     NO_RESPONSE = "0"
 
     """
-    Simple retrieve-then-read implementation, using the Cognitive Search and OpenAI APIs directly. It first retrieves
-    top documents from search, then constructs a prompt with them, and then uses OpenAI to generate an completion
-    (answer) with that prompt.
+    A multi-step approach that first uses ChatGPT to turn the user's question into a search query,
```

Comment:
Should we say GPT instead of ChatGPT, or are we already referring to it like that in other places? I believe the latter is the chat app hosted by Open AI.